src/scripts/Game.ts

```diff
@@ -113,7 +46,7 @@ class Game {
         Underground.counter += GameConstants.TICK_TIME;
 
         GameHelper.counter += GameConstants.TICK_TIME;
-        switch (Game.gameState()) {
+        switch (App.game.gameState) {
```

Comment:
Should this switch be `this.gameState` instead? I know it would have the same effect (unless we have multiple Games running...), but it seems strange that Game knows about App.

Reply:
Ah yes, a remainder from that I did the refactor in 2 steps, it was static before